content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -44,6 +44,18 @@ to the name of the namespace. You can't change this label's value.
 This is useful if you want to target a specific namespace with a label
 {{< glossary_tooltip text="selector" term_id="selector" >}}.
 
+## kubernetes.io/limit-ranger
+
+Example: `kubernetes.io/limit-ranger: LimitRanger plugin set: cpu, memory request for container nginx; cpu, memory limit for container nginx`
+
+Used on: Namespace
+
+Kubernetes by default doesn't provide any resource limit, that means unless you explicitly define limits,
+your container can consume unlimited CPU and memory.
+We can define default limit for pods by creating a LimitRange in the relevant namespace.
+Pods deployed after this LimitRange will have these limits applied to them.
+`limit-ranger` automatically adds the limits that were requested for the pod.
```

Comment:
What is "limit-ranger" ? This sentence is unclear.

Reply:
ok, I will try to make sentence more meaningful.